Algorithms/Medium/142_LinkedListCycleII/Solution.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Definition for singly-linked list.
+ * struct ListNode {
+ *     int val;
+ *     ListNode *next;
+ *     ListNode(int x) : val(x), next(NULL) {}
+ * };
+ */
+class Solution
+{
+public:
+    ListNode *detectCycle(ListNode *head)
+    {
+        if (!head)
+        {
+            printf("no cycle\n");
+            return NULL;
+        }
```

Comment:
Add a blank line to separate the **if** block from the initialization statement.